mx.sulong/mx_sulong.py

```diff
@@ -755,7 +762,7 @@ def getLLVMVersion(llvmProgram):
     versionString = getVersion(llvmProgram)
     printLLVMVersion = re.search(r'(clang |LLVM )?(version )?(3\.\d)', versionString, re.IGNORECASE)
     if printLLVMVersion is None:
-        exit("could not find the LLVM version string in " + str(versionString))
+        return None
```

Comment:
You already opened a PR for this: https://github.com/graalvm/sulong/pull/435. Can you close it if you have this PR here which contains the commits of the other PR?